ext/systemd/puppet.service

```diff
@@ -6,6 +6,8 @@ After=basic.target network.target puppetmaster.service
 [Service]
 EnvironmentFile=-/etc/sysconfig/puppetagent
 EnvironmentFile=-/etc/sysconfig/puppet
+EnvironmentFile=-/etc/default/puppetagent
```

Comment:
The puppetagent variations are for compatibility with older fedora packages. It shouldn't be needed for debian.